tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.PublicApiHasNotChanged.verified.txt

```diff
@@ -378,7 +378,6 @@ namespace Datadog.Trace.ExtensionMethods
 {
     public static class SpanExtensions
     {
-        public static void AddTagsFromDbCommand(this Datadog.Trace.ISpan span, System.Data.IDbCommand command) { }
```

Comment:
I thought about removing this the last time I was here, but I thought it could be a nice helper for users of manual instrumentation.